src/DocScan/DocScanClient.php

```diff
@@ -184,4 +186,34 @@ public function putIbvInstructions(string $sessionId, Instructions $instructions
     {
         $this->docScanService->putIbvInstructions($sessionId, $instructions);
     }
+
```

Comment:
Would be good to add some more information about these functions for clients, so they can see more information about them when they're looking at the function. I'll leave the ones from Java as suggetstions below: